deploy/13_transfer_to_multisig.ts

```diff
@@ -25,6 +33,15 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   if ((await vote.callStatic.owner()) !== multisig.address) {
     await executeAndWait(vote.transferOwnership(multisig.address));
   }
+  if ((await groupHealth.callStatic.owner()) !== multisig.address) {
```

Comment:
this should no longer be needed, if ownership is transfered during deployment.

Reply:
Why not ? This is being used in all other contracts

Reply:
this should only be used when we have to set some dependencies right after deployment. For contract that do not require us to set dependencies, the owner should be set to `multisig` during deployment.

Reply:
I see, thanks, updated :-)